lib/ChangeTypes.js

```diff
@@ -180,6 +180,8 @@ module.exports = {
   SIMPLIFY_RIGHT_SIDE: 'SIMPLIFY_RIGHT_SIDE',
   // e.g. x + 3 = 2 -> x + 3 - 3 = 2 - 3
   SUBTRACT_FROM_BOTH_SIDES: 'SUBTRACT_FROM_BOTH_SIDES',
+  // e.g. x^2 = 1, nthRoot(x^2) = nthRoot(1)
+  NTH_ROOT_BOTH_SIDES: 'NTH_ROOT_BOTH_SIDES',
```

Comment:
Hmm I actually think the plus/minus might be more complicated ... I ran into it when I was thinking about using the quadratic equation also. It might be tricky cause mathjs doesn't support it. I'm thinking maybe we could use a custom function? Let's talk about it.